seer_pas_sdk/core/unsupported.py

```diff
@@ -1448,3 +1448,348 @@ def _get_msdataindex_path(self, display_path: list):
+        report_results["Protein Group"] = report_results["Protein.Group"]
+        report_results["Peptide"] = report_results["Stripped.Sequence"]
+
+        if analyte_type == "protein":
+            report_results = report_results[
+                [
+                    "File Name",
+                    "Protein Group",
+                    "Protein.Ids",
+                    "Global.PG.Q.Value",
+                    "Lib.PG.Q.Value",
+                ]
+            ]
+            report_results.drop_duplicates(
+                subset=["File Name", "Protein Group"], inplace=True
+            )
+            df = pd.merge(
+                search_results,
+                report_results,
+                on=["File Name", "Protein Group"],
+                how="left",
+            )
+        elif analyte_type == "peptide":
+            peptide_results = self.get_search_result(
+                analysis_id=analysis_id, analyte_type="peptide", rollup="np"
+            )
+            peptide_results["File Name"] = peptide_results["File Name"].apply(
+                lambda x: os.path.basename(x).split(".")[0]
+            )
+            peptide_results = peptide_results[
+                ["File Name", "Peptide", "Protein Group"]
+            ]
+            search_results = pd.merge(
+                peptide_results,
+                search_results,
+                on=["File Name", "Protein Group"],
+                how="left",
+            )
+
+            report_results = report_results[
+                [
+                    "File Name",
+                    "Peptide",
+                    "Protein.Ids",
+                ]
+            ]
+            report_results.drop_duplicates(
+                subset=["File Name", "Peptide"], inplace=True
+            )
+            df = pd.merge(
+                search_results,
+                report_results,
+                on=["File Name", "Peptide"],
+                how="left",
+            )
+        else:
+            # precursor
+            search_results = search_results[
+                [
+                    "File Name",
+                    "Plate ID",
+                    "Well",
+                    "Nanoparticle",
+                    "Protein Group",
+                    "Protein Names",
+                    "Gene Names",
+                    "Biological Process",
+                    "Molecular Function",
+                    "Cellular Component",
+                ]
+            ]
+            report_results = report_results[
+                [
+                    "File Name",
+                    "Precursor.Id",
+                    "Peptide",
+                    "Protein Group",
+                    "Protein.Ids",
+                    "Global.Q.Value",
+                    "Lib.Q.Value",
+                ]
+            ]
+            df = pd.merge(
+                search_results,
+                report_results,
+                on=["File Name", "Protein Group"],
```

Comment:
As mentioned before, "File Name" should not be a part of the report.

```suggestion
                on=["Protein Group"],
```